src/edu/ucsb/cs56/projects/games/states_quiz/FrontPanel.java

```diff
@@ -27,8 +27,13 @@
  */
 
 public class FrontPanel extends JPanel {
+<<<<<<< HEAD
```

Comment:
This is junk left over from a merge conflict.  You'll need to resolve the conflict and commit the fixed version.